### ISSUE TITLE
Is it possible to remove horizontal padding?

### ISSUE THREAD

**Issue by hallazzang** [utterance/utterances#209]
It seems there's an unnecessary(IMO) horizontal padding on each side of the rendered comment box:

https://github.com/utterance/utterances/blob/6aec0b7607340748daa9394681c523388f86bf68/src/stylesheets/timeline.scss#L3

Currently this slightly breaks my blog's layout. Is there any option to remove this padding?

**Comment by Gninoskcaj** [comment]
@hallazzang, In your stylesheet.

```css
.timeline {
    padding: 0 !important;
}
```

**Comment by hallazzang** [comment]
@Gninoskcaj Unfortunately It seems impossible to change style of the elements inside an `iframe`.

**Comment by Gninoskcaj** [comment]
@hallazzang You are right, JavaScript is needed to change the style of an element in an `iframe`. My apologies.

```js
const timeline = document.getElementByClassName('timeline');
timeline.style.padding = '0 !important'

**Comment by jdanyow** [comment]
@hallazzang you you could add the following to your site's css to counteract the default padding:

```css
.utterances {
  margin-left: -4px;
  margin-right: -4px;
}
```

You wont be able to modify the iframe's css because it's cross origin, the browser will block any attempt to interact with it's dom from your page's javascript.

**Comment by hallazzang** [comment]
@jdanyow No, it doesn't work either. It makes utterances frame to move left by 4px, rather than make it 4px wider on each side.
Please see this screenshot:

![utterances-padding-issue](https://user-images.githubusercontent.com/5072682/68907482-49e7d480-078b-11ea-955f-0c51f2833fce.png)

But I think we're getting closer to the solution.
